### ISSUE TITLE
Fullscreen (F11) & Hide (H) Not working

### ISSUE THREAD

**Issue by darkstudioteam** [cassidoo/todometer#45]
Not working
---
1. Hide
2. Fullscreen

#### Windows 10 64bit

**Comment by TheDeployGuy** [comment]
Hey guys, I cloned the repo and tried changing (fullscreenable: true)
`  mainWindow = new BrowserWindow({
    width: 800,
    minWidth: 800,
    height: 600,
    fullscreenable: **true**,
    backgroundColor: '#403F4D',
    icon: path.join(__dirname, 'assets/png/128x128.png')
  });`

And now full screen works on windows.(Don't have mac so cannot test on OSX). However, it still shows the menu like so:
![fullscreen](https://user-images.githubusercontent.com/3710874/28247843-f3c18208-6a30-11e7-8c0a-343e95ed27a4.PNG)

@cassidoo if you have already fixed this sorry for relooking :)

**Comment by cassidoo** [comment]
Thanks for looking into it!! And sorry for being slow to reply here.
I have a commit almost ready to go for this part, just have to figure out the Hide function. Might have to be a separate release.

**Comment by TheDeployGuy** [comment]
@cassidoo I looked into the Hide function, I found this on the electron menu wiki:

> macOS has provided standard actions for some menu items, like About xxx, Hide xxx, and Hide Others. To set the action of a menu item to a standard action, you should set the role attribute of the menu item.

I am taking that to mean, the standard `{role: 'hide'}` does nothing on Windows which from my testing is true. I modified it to simply run `mainWindow.hide()` when it is clicked and now it is working. I can submit a PR for it if you like?

One thing I opened #48 which will put a taskbar icon in windows when minimized/closed like OS X. Until that is implemented when you 'hide' the application on windows there is no way to restore the window. I have the code ready to go for both issues, once fully tested will I can go ahead and submit a PR?

Jason
